### ISSUE TITLE
[Java] input<any> widgets do not refresh the buffer provided.

### ISSUE THREAD

**Issue by VIad** [kotlin-graphics/imgui#64]
I have a char buffer that is supposed to be filled with the data from ```inputText()``` function, however when i query the contents, it is all empty.
I've tried using ```inputInt``` as well, same result, the numbers do not change

Here's a short snippet of my code
 ```java
private static char[]	name				= new char[100];

	static void run(ImGui ui) {
		if (ui.begin("New Layer Config", ClawMenuBar.layerCreationDialog, 0)) {
			ui.text("Create new Layer");

			ui.text("Layer Type");
			ui.radioButton("2D", layerType, 0);
			ui.sameLine(60);
			ui.radioButton("3D", layerType, 1);
			if (layerType[0] == 0) {
				ui.dragFloat2("Projection size", newLayerDims.getArray(), .5f, 0f, 3840, "%.2f", 1f);
				ui.colorEdit3("Layer Background", newLayerBackground.getArray(), 0);
				ui.sliderFloat("Layer Opacity", newLayerVisibility, 0f, 1f, "%.3f", 1f);

				ui.inputText("Label", name, 0);
				System.out.println(new String(name));
		       }
		}
	}
```

The output line is always blank, regardless of the widget's text content.
![prob](https://user-images.githubusercontent.com/25646553/47649720-c9a5ff80-db86-11e8-8845-a54432af7b5c.png)

**Comment by elect86** [comment]
Shall be fixed now: e824187ce5

Let me know

**Comment by Sylvyrfysh** [comment]
Hi! Friendly reminder that there has been no activity here for over a month and this appears to be fixed. If you wish to keep this open, please respond with what still is an issue, or this will be closed in a week. Thanks!
